### PR TITLE
ramips: samknows whitebox v8: set wifi frequency

### LINKED ISSUE #15391: ramips/mt7621: SamKnows Whitebox 8: both phy0 and phy1 identified as 5Ghz
### Describe the bug

/etc/config/wireless have both radio0 and radio1 set to band "5g".
iw phy phy0 info shows support for 5GHz .
iw phy phy1 info shows support for 5GHz AND 2.4GHz.
DTS at: target/linux/ramips/dts/mt7621_samknows_whitebox-v8.dts

phy1 does not support 5GHz in reality.

### OpenWrt version

master (r26182-49a64c9541)

### OpenWrt release

master (SNAPSHOT)

### OpenWrt target/subtarget

ramips/mt7621

### Device

SamKnows Whitebox 8

### Image kind

Self-built image

### Steps to reproduce

_No response_

### Actual behaviour

_No response_

### Expected behaviour

There should be a wireless radio configured by default to 2.4GHz.

### Additional info

_No response_

### Diffconfig

_No response_

### Terms

- [X] I am reporting an issue for OpenWrt, not an unsupported fork.

### PR DESCRIPTION
**Pull request by namiltd**
Set the 2.4GHz frequency for WiFi.
Fixes: #15391

### CODE REVIEW

**Review by nunojpg — APPROVED** [review]

File: target/linux/ramips/dts/mt7621_samknows_whitebox-v8.dts [hunk 1]
```diff
@@ -129,6 +129,7 @@
 		reg = <0x0000 0 0 0 0>;
 		nvmem-cells = <&eeprom_factory_0>;
 		nvmem-cell-names = "eeprom";
+		ieee80211-freq-limit = <2400000 2500000>;
```
Comment (hauke) [inline target/linux/ramips/dts/mt7621_samknows_whitebox-v8.dts:132]: This is not needed, it uses a MT7603E which only support 2.4GHz.

Comment (namiltd) [reply target/linux/ramips/dts/mt7621_samknows_whitebox-v8.dts:132]: Look at issue https://github.com/openwrt/openwrt/issues/15391

Comment (nunojpg) [reply target/linux/ramips/dts/mt7621_samknows_whitebox-v8.dts:132]: @hauke if you think the problem has a deeper origin I can test a alternative fix.

**Comment by namiltd** [comment]
@hauke Please also merge https://github.com/openwrt/openwrt/pull/15396

**Pull request merged**